.github/workflows/publish-npm-packages.yml

```
@@ -99,6 +100,15 @@ jobs:
               env:
                   NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
 
+            - name: Publish latest packages to npm ("latest" dist-tag)
```

Would it bring more clarity?

```suggestion
            - name: Publish packages based on the latest Gutenberg plugin to npm ("latest" dist-tag)
```